docs/docs/tutorial/index.md

```diff
@@ -473,9 +443,7 @@ This opens a duplex connection and we should see something like:
 }
 ```
 
-Before continuing, make sure that you see a `PeerConnected` message for each of
-the configured other `hydra-node`. If this is not showing up, double-check
-network configuration and connectivity.
+## Step 4. Open a Hydra head
```

Comment:
This is a duplicate headline to the one below!

Also: do we want to really change the headline from `:` to `.`? If yes, all of them should change consistently.

Reply:
Our style guidelines typically follow the format 'Step x. Start with a capital' or 'Step x: use lowercase', with a preference for the first option. I believe this line is the only inconsistency. @oduameh, please address this by deleting one of the duplicated headlines (the one with `:`).

Reply:
Updated globally